### PR TITLE
Fix links made with annotations

### LINKED ISSUE #91: Links show an overlayed big pin icon
Links in the document are overlayed by a big black pin icon:

image

The icon shows an "Annotation" tooltip when hovering the mouse over it. Clicking it opens an "Annotation attachment" window with all popup menus greyed out.

### PR DESCRIPTION
**Pull request by KapiX**
* Fixes #91.

### CODE REVIEW

**Comment by KapiX** [comment]
Please test with gcc2 before merging, and a few PDF files (I have tested only one).

**Comment by humdingerb** [comment]
Compiles and runs fine with gcc2. All PDFs that show the pin-problem display perfectly pin-less with these changes. Well done!!

File: bepdf/beos/Annotation.cpp [hunk 1]
```diff
@@ -589,6 +589,36 @@ void TextAnnot::Print() {
...
+	// look for destination
+  if (d->lookup("Dest", &obj) && !obj.isNull()) {
+    mLinkAction = LinkAction::parseDest(&obj);
+  // look for action
+  } else {
+    if (d->lookup("A", &obj) && obj.isDict()) {
+      mLinkAction = LinkAction::parseAction(&obj/*, baseURI*/);
+    }
+  }
+  obj.free();
```
Comment (waddlesplash) [inline bepdf/beos/Annotation.cpp]: The rest of the file is in tabs, but this is in spaces. Koder bug? :p

Comment (KapiX) [reply bepdf/beos/Annotation.cpp]: 1. I do not see .editorconfig in this repo. Where would the bug be? :)
2. I was more occupied reverting all the removed whitespace to actually notice that indentation is inconsistent. ;)

Comment (waddlesplash) [reply bepdf/beos/Annotation.cpp]: Oh, can't Koder auto-detect the use of tabs in most source files? I thought it did...

Comment (KapiX) [reply bepdf/beos/Annotation.cpp]: It overrides settings only if there is .editorconfig in the repo (and that has a few bugs). There is no auto-detection.

**Pull request merged**